Foundation/Date.swift

```diff
@@ -137,9 +137,15 @@ public struct Date : ReferenceConvertible, Comparable, Equatable {
      */
     public static let distantPast = Date(timeIntervalSinceReferenceDate: -63114076800.0)
     
+    #if swift(>=4.2)
```

Comment:
The `swift` version check should not be needed as there is a branch per version.